MITADS/opensubtitles_exporter.py

```diff
@@ -121,22 +122,19 @@ def parsexmlfile(xml_path, count_file):
 count_file = 0
 
 # Parse 5 files at once
-pool = ThreadPoolExecutor(max_workers=20)
+pool = ProcessPoolExecutor(max_workers=20)
```

Comment:
I'd use it with the `with` statement so you don't have to explicitly wait for it to finish at the end